Add option for filtering by country

jmintb opened the pull request:
This exposes the country parameter for
filtering to the npm package.

This functionality is usefull for various FWF
projects and probably for other projects in
the future.

Commit:
Add option for filtering by country

This exposes the country parameter for
filtering to the npm package.

This functionality is usefull for various FWF
projects and probably for other projects in
the future.

emiln commented on index.js:
```diff
@@ -81,6 +81,19 @@ class StoresQuery {
 
+  /**
+   * Returns only stores in the given country.
+   * Countries are specified using to letter codes,
```
Specifically, they are specified using [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements). I think there is no harm and potential help in mentioning that, so I would recommend amending this doc string with either a mention of or link to the ISO.

jmintb commented:
Good idea, I will add that to the doc string.

Commit:
Specify country code standard

The doc string now specefies what standard
is used to specify countries, this should
help when trying to figure what country
code to use.
